catalog/java/io/material/catalog/transformation/res/layout/cat_transformation_fragment.xml

```diff
@@ -78,7 +78,7 @@
           android:background="?selectableItemBackgroundBorderless"
           android:contentDescription="@string/close_sheet"
           android:scaleType="fitCenter"
-          android:src="@drawable/ic_close_vd_theme_24px"
+          app:srcCompat="@drawable/ic_close_vd_theme_24px"
           android:tint="?attr/colorOnSurface"/>
```

Comment:
nit: Swap these two attributes so the android namespace attributes are before the app ones.